Source/UnitTests/Core/PowerPC/JitArm64/Fres.cpp

```diff
@@ -40,6 +40,7 @@ class TestFres : public JitArm64
     MOV(ARM64Reg::X1, ARM64Reg::X0);
     m_float_emit.FMOV(ARM64Reg::D0, ARM64Reg::X0);
     m_float_emit.FRECPE(ARM64Reg::D0, ARM64Reg::D0);
+    m_float_emit.FMOV(ARM64Reg::X0, ARM64Reg::D0);
```

Comment:
What's the purpose of this? The fres routine already does `m_float_emit.FMOV(ARM64Reg::X0, ARM64Reg::D0)`.

Reply:
I didn't consider it being part of the FRES routine already, thank you!!